src/components/accession2/create/Accession2Create.tsx

```diff
@@ -44,8 +42,39 @@ export default function CreateAccession(): JSX.Element {
   const history = useHistory();
   const snackbar = useSnackbar();
   const [validateFields, setValidateFields] = useState<boolean>(false);
+  const [selectedSeedBank, setSelectedSeedBank] = useState<Facility>();
+  const tz = useLocationTimeZone().get(selectedSeedBank);
```

Comment:
do you think we should use the isEnabled here and pass undefined if not enabled (instead of selected seedbank)? 
i think if someone has never set the timezone we should not be impacted .. so maybe we could leave it as-is here.. just calling this out if we missed it

Reply:
I will add it just in case